docs/user-guide.md

```diff
@@ -4,6 +4,21 @@ To best understand the benefits of Akri and jump into using it, we recommend you
 ## Getting Started
 To get started using Akri, you must first decide what you want to discover and whether Akri current supports a protocol that can be used to discover resources of that type. To see the list of currently supported protocols, see our [roadmap](./roadmap.md).
 
+### Understanding Akri Helm charts
+Akri is most easily deployed with Helm charts.  Helm charts provide convenient packaging and configuration.
+
+Starting in v0.0.36, there will be a Helm chart published in **akri-dev** for each build version.  Each Akri build is verifeid with end-to-end tests on Kuberenetes, K3s, and MicroK8s, but these are likely to be less stable than our Releases.  You can deploy these versions of Akri with this command (note: **akri-dev**):
+```sh
+helm repo add akri-helm-charts https://deislabs.github.io/akri/
+helm install akri akri-helm-charts/akri-dev
+```
+
+After v0.0.36, a Helm chart will be published in **akri** for each release.  Releases will generally reflect milestones and will have more rigorous testing.  You can deploy release versions of Akri with this command (note: **akri**):
```

Comment:
Should this say "in akri <what> for each release" ?

Reply:
do you mean without the emphasis?

Reply:
Sorry, I am not very good with the clarity these days... Also, there was an extra token which got deleted by the rendering. 

`in **akri** <what> for each release `

where `<what>` should be repo/chart/something. Or is it just me that it feels unqualified?

Reply:
I had stream and feed in a few drafts, but figured nothing was just as good.

Happy to change to whatever feels better or to the right word if someone knows it.

Reply:
Would this make sense?

After v0.0.36, **akri** Helm chart will be published for each release.